{"instance_index": 2, "error_index": 1, "failure_commit": "90d5d27fcfbce4bd1253d3220c6a5232d234b40a", "repair_commit": "e7d5398aa1349cc575a5b80e0d6eb28e61cb4bfa", "error_line": "crypto/cms/cms_asn1.c:240:73: error: expected ';' after top level declarator", "error_detail": "crypto/cms/cms_asn1.c:240:73: error: expected ';' after top level declarator\n  240 | } ASN1_ADB_END(CMS_OtherRecipientInfo, 0, oriType, 0, &ori_def_tt, NULL)\n      |                                                                         ^\n      |                                                                         ;", "compilation_related_path": {"error_file": ["crypto/cms/cms_asn1.c"], "direct_includes": ["crypto/cms/cms_local.h"]}, "original_error_lines": ["crypto/cms/cms_asn1.c:240:73: error: expected ';' after top level declarator"], "workflow_name": "GitHub CI", "job_name": ["basic_clang", "no-shared-macos (macos-13)", "freebsd-x86_64", "threads_sanitizer", "no-shared-macos (macos-14)", "memory_sanitizer", "out-of-readonly-source-and-install-macos (macos-13)", "out-of-readonly-source-and-install-macos (macos-14)"]}
{"instance_index": 3, "error_index": 1, "failure_commit": "4bdc1981f96a70c8f97f735e953c075d17840ca1", "repair_commit": "61e1905a7a22f690446df63007e6dccdb53af146", "error_line": "ssl/quic/quic_ackm.c:1037:12: error: conflicting types for 'ossl_ackm_new'; have 'OSSL_ACKM *(OSSL_TIME (*)(void *), void *, OSSL_STATM *, const OSSL_CC_METHOD *, OSSL_CC_DATA *, char)' {aka 'struct ossl_ackm_st *(OSSL_TIME (*)(void *), void *, struct ossl_statm_st *, const struct ossl_cc_method_st *, struct ossl_cc_data_st *, char)'}", "error_detail": " 1037 | OSSL_ACKM *ossl_ackm_new(OSSL_TIME (*now)(void *arg),\n      |            ^~~~~~~~~~~~~", "compilation_related_path": {"error_file": ["ssl/quic/quic_ackm.c"]}, "original_error_lines": ["ssl/quic/quic_ackm.c:1037:12: error: conflicting types for 'ossl_ackm_new'; have 'OSSL_ACKM *(OSSL_TIME (*)(void *), void *, OSSL_STATM *, const OSSL_CC_METHOD *, OSSL_CC_DATA *, char)' {aka 'struct ossl_ackm_st *(OSSL_TIME (*)(void *), void *, struct ossl_statm_st *, const struct ossl_cc_method_st *, struct ossl_cc_data_st *, char)'}"], "workflow_name": "GitHub CI", "job_name": ["check-c99", "basic_gcc", "linux-arm64", "no-shared-ubuntu", "fuzz_tests", "no-legacy", "address_ub_sanitizer", "legacy", "no-deprecated", "enable_non-default_options", "non-caching", "full_featured", "out-of-readonly-source-and-install-ubuntu", "external-tests-misc", "external-tests-pkcs11-provider", "external-tests-oqs-provider", "external-tests-pyca (3.9)"]}
{"instance_index": 5, "error_index": 1, "failure_commit": "e9e56e513ff54e9a3da168845ea94b9e30fcb3e2", "repair_commit": "491fe4c6d06eadb288da5ee0ee75bfb311aade36", "error_line": "ssl/s3_lib.c:4490:5: error: 'alg_k' undeclared (first use in this function); did you mean 'alg_a'?", "error_detail": "ssl/s3_lib.c:4490:5: error: 'alg_k' undeclared (first use in this function); did you mean 'alg_a'?\n 4490 |     alg_k = s->s3.tmp.new_cipher->algorithm_mkey;\n      |     ^~~~~\n      |     alg_a\nssl/s3_lib.c:4490:5: note: each undeclared identifier is reported only once for each function it appears in\nAt top level:\ncc1: note: unrecognized command-line option '-Wno-unterminated-string-initialization' may have been intended to silence earlier diagnostics", "compilation_related_path": {"error_file": ["ssl/s3_lib.c"]}, "original_error_lines": ["ssl/s3_lib.c:4490:5: error: 'alg_k' undeclared (first use in this function); did you mean 'alg_a'?"], "workflow_name": "GitHub CI", "job_name": ["enable_non-default_options", "minimal"]}
{"instance_index": 6, "error_index": 1, "failure_commit": "d98c44f116490189453ebd91b1a0d0a3f1f21041", "repair_commit": "6c108b9e4590903a94347cb572d24bc8ebe2c753", "error_line": "crypto/threads_pthread.c:747:60: error: 'thread_contention_data' undeclared (first use in this function)", "error_detail": "crypto/threads_pthread.c:747:60: error: 'thread_contention_data' undeclared (first use in this function)\n  747 |     struct stack_traces *traces = CRYPTO_THREAD_get_local(&thread_contention_data);\n      |                                                            ^~~~~~~~~~~~~~~~~~~~~~\ncrypto/threads_pthread.c:747:60: note: each undeclared identifier is reported only once for each function it appears in", "compilation_related_path": {"error_file": ["crypto/threads_pthread.c"]}, "original_error_lines": ["crypto/threads_pthread.c:747:60: error: 'thread_contention_data' undeclared (first use in this function)", "crypto/threads_pthread.c:749:40: error: invalid application of 'sizeof' to incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:752:11: error: invalid use of undefined type 'struct stack_traces'", "crypto/threads_pthread.c:792:60: error: 'thread_contention_data' undeclared (first use in this function)", "crypto/threads_pthread.c:794:40: error: invalid application of 'sizeof' to incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:797:11: error: invalid use of undefined type 'struct stack_traces'"], "workflow_name": "GitHub CI", "job_name": ["non-caching", "no-deprecated", "check-c99", "linux-arm64", "address_ub_sanitizer", "minimal", "basic_gcc", "fuzz_tests", "no-shared-ubuntu", "no-legacy", "legacy", "out-of-readonly-source-and-install-ubuntu", "external-tests-oqs-provider", "external-tests-pyca (3.9)", "full_featured", "external-tests-misc", "external-tests-pkcs11-provider"]}
{"instance_index": 6, "error_index": 2, "failure_commit": "d98c44f116490189453ebd91b1a0d0a3f1f21041", "repair_commit": "6c108b9e4590903a94347cb572d24bc8ebe2c753", "error_line": "crypto/threads_pthread.c:749:40: error: invalid application of 'sizeof' to incomplete type 'struct stack_traces'", "error_detail": "In file included from crypto/threads_pthread.c:34:\ncrypto/threads_pthread.c:749:40: error: invalid application of 'sizeof' to incomplete type 'struct stack_traces'\n  749 |         traces = OPENSSL_zalloc(sizeof(struct stack_traces));\n      |                                        ^~~~~~\ninclude/openssl/crypto.h:104:23: note: in definition of macro 'OPENSSL_zalloc'\n  104 |         CRYPTO_zalloc(num, OPENSSL_FILE, OPENSSL_LINE)\n      |                       ^~~", "compilation_related_path": {"error_file": ["crypto/threads_pthread.c"]}, "original_error_lines": ["crypto/threads_pthread.c:747:60: error: 'thread_contention_data' undeclared (first use in this function)", "crypto/threads_pthread.c:749:40: error: invalid application of 'sizeof' to incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:752:11: error: invalid use of undefined type 'struct stack_traces'", "crypto/threads_pthread.c:792:60: error: 'thread_contention_data' undeclared (first use in this function)", "crypto/threads_pthread.c:794:40: error: invalid application of 'sizeof' to incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:797:11: error: invalid use of undefined type 'struct stack_traces'"], "workflow_name": "GitHub CI", "job_name": ["non-caching", "no-deprecated", "check-c99", "linux-arm64", "address_ub_sanitizer", "minimal", "basic_gcc", "fuzz_tests", "no-shared-ubuntu", "no-legacy", "legacy", "out-of-readonly-source-and-install-ubuntu", "external-tests-oqs-provider", "external-tests-pyca (3.9)", "full_featured", "external-tests-misc", "external-tests-pkcs11-provider"]}
{"instance_index": 6, "error_index": 3, "failure_commit": "d98c44f116490189453ebd91b1a0d0a3f1f21041", "repair_commit": "6c108b9e4590903a94347cb572d24bc8ebe2c753", "error_line": "crypto/threads_pthread.c:752:11: error: invalid use of undefined type 'struct stack_traces'", "error_detail": "crypto/threads_pthread.c:752:11: error: invalid use of undefined type 'struct stack_traces'\n  752 |     traces->lock_depth++;\n      |           ^~\ncrypto/threads_pthread.c: In function 'CRYPTO_THREAD_write_lock':", "compilation_related_path": {"error_file": ["crypto/threads_pthread.c"]}, "original_error_lines": ["crypto/threads_pthread.c:747:60: error: 'thread_contention_data' undeclared (first use in this function)", "crypto/threads_pthread.c:749:40: error: invalid application of 'sizeof' to incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:752:11: error: invalid use of undefined type 'struct stack_traces'", "crypto/threads_pthread.c:792:60: error: 'thread_contention_data' undeclared (first use in this function)", "crypto/threads_pthread.c:794:40: error: invalid application of 'sizeof' to incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:797:11: error: invalid use of undefined type 'struct stack_traces'"], "workflow_name": "GitHub CI", "job_name": ["non-caching", "no-deprecated", "check-c99", "linux-arm64", "address_ub_sanitizer", "minimal", "basic_gcc", "fuzz_tests", "no-shared-ubuntu", "no-legacy", "legacy", "out-of-readonly-source-and-install-ubuntu", "external-tests-oqs-provider", "external-tests-pyca (3.9)", "full_featured", "external-tests-misc", "external-tests-pkcs11-provider"]}
{"instance_index": 7, "error_index": 1, "failure_commit": "d98c44f116490189453ebd91b1a0d0a3f1f21041", "repair_commit": "6c108b9e4590903a94347cb572d24bc8ebe2c753", "error_line": "crypto/threads_pthread.c:747:60: error: use of undeclared identifier 'thread_contention_data'", "error_detail": "crypto/threads_pthread.c:747:60: error: use of undeclared identifier 'thread_contention_data'\n    struct stack_traces *traces = CRYPTO_THREAD_get_local(&thread_contention_data);\n                                                           ^", "compilation_related_path": {"error_file": ["crypto/threads_pthread.c"]}, "original_error_lines": ["crypto/threads_pthread.c:747:60: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:749:33: error: invalid application of 'sizeof' to an incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:750:34: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:752:11: error: incomplete definition of type 'struct stack_traces'", "crypto/threads_pthread.c:792:60: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:794:33: error: invalid application of 'sizeof' to an incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:795:34: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:797:11: error: incomplete definition of type 'struct stack_traces'"], "workflow_name": "GitHub CI", "job_name": ["no-shared-macos (macos-14)", "basic_clang", "no-shared-macos (macos-13)", "freebsd-x86_64", "threads_sanitizer", "out-of-readonly-source-and-install-macos (macos-13)", "memory_sanitizer", "out-of-readonly-source-and-install-macos (macos-14)"]}
{"instance_index": 1, "error_index": 1, "failure_commit": "07748a9d4115e208b19470462e76c1ab38b7693c", "repair_commit": "a7b9fa866579a080bc52c6d091f755345907a4d4", "error_line": "crypto/dh/dh_backend.c:85:38: error: too few arguments to function call, expected 2, have 1", "error_detail": "crypto/dh/dh_backend.c:85:38: error: too few arguments to function call, expected 2, have 1\n   85 |         if (ossl_dh_check_pairwise(dh) == 0)\n      |             ~~~~~~~~~~~~~~~~~~~~~~   ^\ninclude/crypto/dh.h:45:5: note: 'ossl_dh_check_pairwise' declared here\n   45 | int ossl_dh_check_pairwise(const DH *dh, int return_on_null_numbers);\n      |     ^                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~", "compilation_related_path": {"direct_includes": ["include/crypto/dh.h"], "indirect_includes": ["include/crypto/dh.h"]}, "original_error_lines": ["crypto/dh/dh_backend.c:85:38: error: too few arguments to function call, expected 2, have 1"], "workflow_name": "GitHub CI", "job_name": ["memory_sanitizer", "freebsd-x86_64", "out-of-readonly-source-and-install-macos (macos-13)", "out-of-readonly-source-and-install-macos (macos-14)"]}
{"instance_index": 7, "error_index": 2, "failure_commit": "d98c44f116490189453ebd91b1a0d0a3f1f21041", "repair_commit": "6c108b9e4590903a94347cb572d24bc8ebe2c753", "error_line": "crypto/threads_pthread.c:749:33: error: invalid application of 'sizeof' to an incomplete type 'struct stack_traces'", "error_detail": "crypto/threads_pthread.c:749:33: error: invalid application of 'sizeof' to an incomplete type 'struct stack_traces'\n        traces = OPENSSL_zalloc(sizeof(struct stack_traces));\n                                ^     ~~~~~~~~~~~~~~~~~~~~~\ninclude/openssl/crypto.h:104:23: note: expanded from macro 'OPENSSL_zalloc'\n        CRYPTO_zalloc(num, OPENSSL_FILE, OPENSSL_LINE)\n                      ^~~\ncrypto/threads_pthread.c:747:12: note: forward declaration of 'struct stack_traces'\n    struct stack_traces *traces = CRYPTO_THREAD_get_local(&thread_contention_data);\n           ^", "compilation_related_path": {"error_file": ["crypto/threads_pthread.c"]}, "original_error_lines": ["crypto/threads_pthread.c:747:60: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:749:33: error: invalid application of 'sizeof' to an incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:750:34: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:752:11: error: incomplete definition of type 'struct stack_traces'", "crypto/threads_pthread.c:792:60: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:794:33: error: invalid application of 'sizeof' to an incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:795:34: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:797:11: error: incomplete definition of type 'struct stack_traces'"], "workflow_name": "GitHub CI", "job_name": ["no-shared-macos (macos-14)", "basic_clang", "no-shared-macos (macos-13)", "freebsd-x86_64", "threads_sanitizer", "out-of-readonly-source-and-install-macos (macos-13)", "memory_sanitizer", "out-of-readonly-source-and-install-macos (macos-14)"]}
{"instance_index": 7, "error_index": 4, "failure_commit": "d98c44f116490189453ebd91b1a0d0a3f1f21041", "repair_commit": "6c108b9e4590903a94347cb572d24bc8ebe2c753", "error_line": "crypto/threads_pthread.c:752:11: error: incomplete definition of type 'struct stack_traces'", "error_detail": "crypto/threads_pthread.c:752:11: error: incomplete definition of type 'struct stack_traces'\n    traces->lock_depth++;\n    ~~~~~~^\ncrypto/threads_pthread.c:747:12: note: forward declaration of 'struct stack_traces'\n    struct stack_traces *traces = CRYPTO_THREAD_get_local(&thread_contention_data);\n           ^", "compilation_related_path": {"error_file": ["crypto/threads_pthread.c"]}, "original_error_lines": ["crypto/threads_pthread.c:747:60: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:749:33: error: invalid application of 'sizeof' to an incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:750:34: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:752:11: error: incomplete definition of type 'struct stack_traces'", "crypto/threads_pthread.c:792:60: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:794:33: error: invalid application of 'sizeof' to an incomplete type 'struct stack_traces'", "crypto/threads_pthread.c:795:34: error: use of undeclared identifier 'thread_contention_data'", "crypto/threads_pthread.c:797:11: error: incomplete definition of type 'struct stack_traces'"], "workflow_name": "GitHub CI", "job_name": ["no-shared-macos (macos-14)", "basic_clang", "no-shared-macos (macos-13)", "freebsd-x86_64", "threads_sanitizer", "out-of-readonly-source-and-install-macos (macos-13)", "memory_sanitizer", "out-of-readonly-source-and-install-macos (macos-14)"]}
{"instance_index": 4, "error_index": 1, "failure_commit": "4bdc1981f96a70c8f97f735e953c075d17840ca1", "repair_commit": "61e1905a7a22f690446df63007e6dccdb53af146", "error_line": "ssl/quic/quic_ackm.c:1037:12: error: conflicting types for 'ossl_ackm_new'", "error_detail": "ssl/quic/quic_ackm.c:1037:12: error: conflicting types for 'ossl_ackm_new'\nOSSL_ACKM *ossl_ackm_new(OSSL_TIME (*now)(void *arg),\n           ^\ninclude/internal/quic_ackm.h:22:12: note: previous declaration is here\nOSSL_ACKM *ossl_ackm_new(OSSL_TIME (*now)(void *arg),\n           ^", "compilation_related_path": {"error_file": ["ssl/quic/quic_ackm.c"]}, "original_error_lines": ["ssl/quic/quic_ackm.c:1037:12: error: conflicting types for 'ossl_ackm_new'"], "workflow_name": "GitHub CI", "job_name": ["no-shared-macos (macos-13)", "no-shared-macos (macos-14)", "memory_sanitizer", "basic_clang", "threads_sanitizer", "freebsd-x86_64", "out-of-readonly-source-and-install-macos (macos-14)", "out-of-readonly-source-and-install-macos (macos-13)"]}
{"instance_index": 8, "error_index": 1, "failure_commit": "a17e2d9c6d2b6741778a80a80a85be84d4b0e9a6", "repair_commit": "de5a619aa015e7c8648e415975e5e2b722b2cbf7", "error_line": "apps/s_client.c:945:5: error: unknown type name 'bool'", "error_detail": "apps/s_client.c:945:5: error: unknown type name 'bool'\n  945 |     bool record_size_limit_set = false;\n      |     ^~~~\napps/s_client.c:51:1: note: 'bool' is defined in header '<stdbool.h>'; did you forget to '#include <stdbool.h>'?\n   50 | #include \"timeouts.h\"\n  +++ |+#include <stdbool.h>\n   51 | #include \"internal/sockets.h\"", "compilation_related_path": {"error_file": ["apps/s_client.c"]}, "original_error_lines": ["apps/s_client.c:945:5: error: unknown type name 'bool'", "apps/s_client.c:945:34: error: 'false' undeclared (first use in this function)", "apps/s_client.c:1553:37: error: 'true' undeclared (first use in this function)"], "workflow_name": "GitHub CI", "job_name": ["minimal"]}
{"instance_index": 10, "error_index": 1, "failure_commit": "4b6f6604d8ad09c1fc0dcd3f122fd6dfa4c02ace", "repair_commit": "e1bccde1a76cb3c2b9204093083b8704b81ac37f", "error_line": "ssl/record/rec_layer_d1.c:618:26: error: 'sc' undeclared (first use in this function); did you mean 's'?", "error_detail": "In file included from ssl/record/../ssl_local.h:31,\nssl/record/rec_layer_d1.c:618:26: error: 'sc' undeclared (first use in this function); did you mean 's'?\n  618 |                 SSLfatal(sc, SSL_AD_HANDSHAKE_FAILURE, SSL_R_NO_RENEGOTIATION);\n      |                          ^~\nssl/record/../statem/statem.h:139:44: note: in definition of macro 'SSLfatal'\n  139 | # define SSLfatal(s, al, r) SSLfatal_data((s), (al), (r), NULL)\n      |                                            ^\nssl/record/rec_layer_d1.c:618:26: note: each undeclared identifier is reported only once for each function it appears in\n  618 |                 SSLfatal(sc, SSL_AD_HANDSHAKE_FAILURE, SSL_R_NO_RENEGOTIATION);\n      |                          ^~\nssl/record/../statem/statem.h:139:44: note: in definition of macro 'SSLfatal'\n  139 | # define SSLfatal(s, al, r) SSLfatal_data((s), (al), (r), NULL)\n      |                                            ^\nAt top level:\ncc1: note: unrecognized command-line option '-Wno-unterminated-string-initialization' may have been intended to silence earlier diagnostics", "compilation_related_path": {"error_file": ["ssl/record/rec_layer_d1.c"]}, "original_error_lines": ["ssl/record/rec_layer_d1.c:618:26: error: 'sc' undeclared (first use in this function); did you mean 's'?"], "workflow_name": "GitHub CI", "job_name": ["basic_gcc", "check-ansi", "linux-arm64", "minimal", "no-deprecated", "address_ub_sanitizer", "buildtest", "enable_non-default_options", "fips_and_ktls", "legacy", "no-legacy", "non-caching", "no-shared-ubuntu", "out-of-readonly-source-and-install-ubuntu", "external-test-pyca (1.51.0, 3.9)", "external-tests"]}
{"instance_index": 8, "error_index": 2, "failure_commit": "a17e2d9c6d2b6741778a80a80a85be84d4b0e9a6", "repair_commit": "de5a619aa015e7c8648e415975e5e2b722b2cbf7", "error_line": "apps/s_client.c:945:34: error: 'false' undeclared (first use in this function)", "error_detail": "apps/s_client.c:945:34: error: 'false' undeclared (first use in this function)\n  945 |     bool record_size_limit_set = false;\n      |                                  ^~~~~\napps/s_client.c:945:34: note: 'false' is defined in header '<stdbool.h>'; did you forget to '#include <stdbool.h>'?\napps/s_client.c:945:34: note: each undeclared identifier is reported only once for each function it appears in", "compilation_related_path": {"error_file": ["apps/s_client.c"]}, "original_error_lines": ["apps/s_client.c:945:5: error: unknown type name 'bool'", "apps/s_client.c:945:34: error: 'false' undeclared (first use in this function)", "apps/s_client.c:1553:37: error: 'true' undeclared (first use in this function)"], "workflow_name": "GitHub CI", "job_name": ["minimal"]}
{"instance_index": 9, "error_index": 1, "failure_commit": "4b6f6604d8ad09c1fc0dcd3f122fd6dfa4c02ace", "repair_commit": "e1bccde1a76cb3c2b9204093083b8704b81ac37f", "error_line": "ssl/record/rec_layer_d1.c:618:26: error: use of undeclared identifier 'sc'", "error_detail": "ssl/record/rec_layer_d1.c:618:26: error: use of undeclared identifier 'sc'\n  618 |                 SSLfatal(sc, SSL_AD_HANDSHAKE_FAILURE, SSL_R_NO_RENEGOTIATION);\n      |                          ^", "compilation_related_path": {"error_file": ["ssl/record/rec_layer_d1.c"]}, "original_error_lines": ["ssl/record/rec_layer_d1.c:618:26: error: use of undeclared identifier 'sc'"], "workflow_name": "GitHub CI", "job_name": ["basic_clang", "freebsd-x86_64", "memory_sanitizer", "no-shared-macos (macos-13)", "no-shared-macos (macos-14)", "threads_sanitizer", "out-of-readonly-source-and-install-macos (macos-13)", "out-of-readonly-source-and-install-macos (macos-14)"]}
{"instance_index": 12, "error_index": 1, "failure_commit": "322872068199942a581d26ed72e6dde6ac2d75f2", "repair_commit": "7cc69ec4cf575232040ac776f405fb31f030e59a", "error_line": "ssl/s3_lib.c:5044:22: error: 'PROV_R_FAILED_DURING_DERIVATION' undeclared (first use in this function)", "error_detail": "In file included from ssl/ssl_local.h:30,\nssl/s3_lib.c:5044:22: error: 'PROV_R_FAILED_DURING_DERIVATION' undeclared (first use in this function)\n 5044 |                      PROV_R_FAILED_DURING_DERIVATION);\n      |                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\ninclude/internal/statem.h:148:55: note: in definition of macro 'SSLfatal'\n  148 | # define SSLfatal(s, al, r) SSLfatal_data((s), (al), (r), NULL)\n      |                                                       ^\nssl/s3_lib.c:5044:22: note: each undeclared identifier is reported only once for each function it appears in\n 5044 |                      PROV_R_FAILED_DURING_DERIVATION);\n      |                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\ninclude/internal/statem.h:148:55: note: in definition of macro 'SSLfatal'\n  148 | # define SSLfatal(s, al, r) SSLfatal_data((s), (al), (r), NULL)\n      |                                                       ^\nAt top level:\ncc1: note: unrecognized command-line option '-Wno-unterminated-string-initialization' may have been intended to silence earlier diagnostics", "compilation_related_path": {"error_file": ["ssl/s3_lib.c"]}, "original_error_lines": ["ssl/s3_lib.c:5044:22: error: 'PROV_R_FAILED_DURING_DERIVATION' undeclared (first use in this function)"], "workflow_name": "GitHub CI", "job_name": ["check-ansi", "basic_gcc", "linux-arm64", "minimal", "no-deprecated", "no-shared-ubuntu", "address_ub_sanitizer", "enable_non-default_options", "full_featured", "fuzz_tests", "legacy", "no-legacy", "non-caching", "out-of-readonly-source-and-install-ubuntu", "external-tests-misc", "external-tests-providers", "external-tests-pyca (3.9)"]}
{"instance_index": 11, "error_index": 1, "failure_commit": "c489a6205ec25528d4ab692694571af75b35879e", "repair_commit": "f32e8a48d20c448a6706e63ce920afbc4b40b203", "error_line": "include/internal/numbers.h:109:1: error: C++ style comments are not allowed in ISO C90", "error_detail": "In file included from include/internal/e_os.h:18,\ninclude/internal/numbers.h:109:1: error: C++ style comments are not allowed in ISO C90\n  109 | // Fix for cross compiling 64-bit PowerPC on OS X 10.4\n      | ^", "compilation_related_path": {"error_file": ["include/internal/numbers.h"]}, "original_error_lines": ["include/internal/numbers.h:109:1: error: C++ style comments are not allowed in ISO C90"], "workflow_name": "GitHub CI", "job_name": ["check-ansi"]}
{"instance_index": 8, "error_index": 3, "failure_commit": "a17e2d9c6d2b6741778a80a80a85be84d4b0e9a6", "repair_commit": "de5a619aa015e7c8648e415975e5e2b722b2cbf7", "error_line": "apps/s_client.c:1553:37: error: 'true' undeclared (first use in this function)", "error_detail": "apps/s_client.c:1553:37: error: 'true' undeclared (first use in this function)\n 1553 |             record_size_limit_set = true;\n      |                                     ^~~~\napps/s_client.c:1553:37: note: 'true' is defined in header '<stdbool.h>'; did you forget to '#include <stdbool.h>'?\nAt top level:\ncc1: note: unrecognized command-line option '-Wno-unterminated-string-initialization' may have been intended to silence earlier diagnostics", "compilation_related_path": {"error_file": ["apps/s_client.c"]}, "original_error_lines": ["apps/s_client.c:945:5: error: unknown type name 'bool'", "apps/s_client.c:945:34: error: 'false' undeclared (first use in this function)", "apps/s_client.c:1553:37: error: 'true' undeclared (first use in this function)"], "workflow_name": "GitHub CI", "job_name": ["minimal"]}
{"instance_index": 13, "error_index": 1, "failure_commit": "322872068199942a581d26ed72e6dde6ac2d75f2", "repair_commit": "7cc69ec4cf575232040ac776f405fb31f030e59a", "error_line": "ssl/s3_lib.c:5044:22: error: use of undeclared identifier 'PROV_R_FAILED_DURING_DERIVATION'", "error_detail": "ssl/s3_lib.c:5044:22: error: use of undeclared identifier 'PROV_R_FAILED_DURING_DERIVATION'\n 5044 |                      PROV_R_FAILED_DURING_DERIVATION);\n      |                      ^", "compilation_related_path": {"error_file": ["ssl/s3_lib.c"]}, "original_error_lines": ["ssl/s3_lib.c:5044:22: error: use of undeclared identifier 'PROV_R_FAILED_DURING_DERIVATION'"], "workflow_name": "GitHub CI", "job_name": ["basic_clang", "freebsd-x86_64", "no-shared-macos (macos-13)", "no-shared-macos (macos-14)", "memory_sanitizer", "out-of-readonly-source-and-install-macos (macos-13)", "out-of-readonly-source-and-install-macos (macos-14)", "threads_sanitizer"]}
